services/client/src/styles/_detail.scss

```
@@ -72,101 +155,107 @@
+$color-tag-user-bg: #dcecf8;
+
 .tag {
-    display: inline-block;
-    padding: 0.2rem 0.6rem;
-    border-radius: 999px;
-    font-size: 0.78rem;
-    font-weight: 500;
-    background: $color-bg-tag-user;
-    color: $color-primary;
+    @extend %pill;
+    background: $color-tag-user-bg;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)